config/initializers/content_security_policy.rb

```diff
@@ -5,30 +5,23 @@
 # For further information see the following documentation
 # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
 
-# Rails.application.config.content_security_policy do |policy|
-#   policy.default_src :self, :https
-#   policy.font_src    :self, :https, :data
-#   policy.img_src     :self, :https, :data
-#   policy.object_src  :none
-#   policy.script_src  :self, :https
-#   policy.style_src   :self, :https
-#   # If you are using webpack-dev-server then specify webpack-dev-server host
-#   if Rails.env.development?
-#     policy.connect_src :self, :https, "http://localhost:3035", "ws://localhost:3035"
```

Comment:
I think this was removed by accident.
We want this to avoid problems with webpack-dev-server in development.

Reply:
Since this was all commented out, I thought we hadn't made changes to the defaults.

I'll re-add the webpack-dev-server-part, and have a look in the git logs to see if we made other changes to the file that we want to keep.

Reply:
It turns out that the webpacker-specific code has been moved from rails to webpacker:
https://github.com/rails/rails/commit/82e44320589fcbacf16012fd7fdca07835b88f20#diff-a8088f5673c46f1598d74d8502f881f005063142132fe7e42d6ae1d2c6bca190L14
https://github.com/rails/webpacker/blob/550c53eb7816140a24b3af9691fbbdf23f24a563/lib/install/template.rb#L57

Unfortunately it seems to have the side-effect that the rails upgrade script no longer includes it.

Reply:
This has now been fixed in main